mlir/include/mlir/Dialect/X86Vector/X86VectorInterfaces.td

```diff
@@ -14,57 +14,18 @@
 #define X86VECTOR_INTERFACES
 
 include "mlir/IR/Interfaces.td"
+include "mlir/Dialect/LLVMIR/LLVMInterfaces.td"
 
 //===----------------------------------------------------------------------===//
-// One-to-One Intrinsic Interface
+// Intrinsic Interface
 //===----------------------------------------------------------------------===//
 
-def OneToOneIntrinsicOpInterface : OpInterface<"OneToOneIntrinsicOp"> {
+def IntrinsicOpInterface
+    : OpInterface<"IntrinsicOp", [OneToOneIntrinsicOpInterface]> {
```

Comment:
[nit] I would include X86 in the name - to optimise for "grep"ing.

Reply:
Why is the wrapper interface necessary? Do we want to mark x86 intrinsics specifically?

Reply:
For now yes, I intend to specifically target only x86 and later only amx ops in their respective legalization passes.
To be fair, so far the two conversions (after initial testing) are identical and we could probably get away with a single generic converter. But I want to keep these separate for now, see how that holds up, and maybe refactor later.

Reply:
Ok, sounds good to me.